mne/viz/evoked.py

```diff
@@ -1198,3 +1199,437 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
+            ymin_ = ymin
+        if ymax is not None and ymax < ymax_:
+            ymax_ = ymax
+        axes.set_yticks((ymin_ if any_negative else 0,
+                         ymax_ if any_positive else 0))
+        ymin_bound, ymax_bound = (-(abs_lims // fraction),
+                                  abs_lims // fraction)
+        # user supplied ymin and ymax overwrite everything
+        if ymin is not None and ymin > ymin_bound:
+            ymin_bound = ymin
+        if ymax is not None and ymax < ymax_bound:
+            ymax_bound = ymax
+        precision = 0.25  # round to .25
+        if ymin is None:
+            ymin_bound = round(ymin_bound / precision) * precision
+        if ymin is None:
+            ymax_bound = round(ymax_bound / precision) * precision
+        axes.spines['left'].set_bounds(ymin_bound, ymax_bound)
+    else:
+        if ymin is not None and ymin > 0:
+            warn("ymin is positive, not truncating yaxis")
+        ymax_bound = axes.get_ylim()[-1]
+    y_range = -np.subtract(*axes.get_ylim())
+
+    title = ", ".join(ch_names[:6]) if title is None else title
+    if len(ch_names) > 6 and gfp is False:
+        warn("More than 6 channels, truncating title ...")
+        title += ", ..."
+    axes.set_title(title)
+
+    # style the spines/axes
+    axes.spines["top"].set_position('zero')
+    axes.spines["top"].set_smart_bounds(True)
+
+    axes.tick_params(direction='out')
+    axes.tick_params(right="off")
+
+    current_ymin = axes.get_ylim()[0]
+
+    # plot v lines
+    if invert_y is True and current_ymin < 0:
+        upper_v, lower_v = -ymax_bound, axes.get_ylim()[-1]
+    else:
+        upper_v, lower_v = axes.get_ylim()[0], ymax_bound
+    axes.vlines(vlines, upper_v, lower_v, linestyles='--', colors='k',
+                linewidth=1., zorder=10)
+
+    # set x label
+    axes.set_xlabel('Time (s)')
+    axes.xaxis.get_label().set_verticalalignment('center')
+
+    # set y label and ylabel position
+    axes.set_ylabel(_handle_default("units")[ch_type], rotation=0)
+    ylabel_height = (-(current_ymin / y_range)
+                     if 0 > current_ymin  # ... if we have negative values
+                     else (axes.get_yticks()[-1] / 2 / y_range))
+    axes.yaxis.set_label_coords(-0.05, 1 - ylabel_height
+                                if invert_y else ylabel_height)
+    xticks = sorted(list(set([x for x in axes.get_xticks()] + vlines)))
+    axes.set_xticks(xticks)
+    axes.set_xticklabels(xticks)
+    x_extrema = [t for t in xticks if tmax >= t >= tmin]
+    axes.spines['bottom'].set_bounds(x_extrema[0], x_extrema[-1])
+    axes.spines["left"].set_zorder(1)
+
+    # finishing touches
+    if invert_y:
+        axes.invert_yaxis()
+    axes.patch.set_alpha(0)
+    axes.spines['right'].set_color('none')
+    axes.set_xlim(tmin, tmax)
+
+    if len(conditions) > 1:
+        plt.legend(loc='best', ncol=1 + (len(conditions) // 5),
+                   frameon=True, prop=dict(size=10))
```

Comment:
Good point, it should be seaborn compliant. (It only partially is, because Seaborn does its own despining. It looks okay with sns.set_style("white"), but lacks the ticks.)
I'll check.